### PR TITLE
storybook(fxa-settings): Recreate ConfirmResetPassword in React

### PR DESCRIPTION
**Pull request by vpomerleau**
## Because

- We want to convert content-server views to React and are starting by building out the front-end in Storybook. This PR converts the confirm_reset_password view. Additional complexity will be added in (a) later ticket(s).

## This pull request

- Create a new ConfirmPasswordPage in React, with Storybook, Fluent file, and initial metrics and tests.
- Update FTL comment in ResetPassword for improved clarity for localizers.

## Issue that this pull request solves

Closes: FXA-6339

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

New component in Storybook

image

## Other information (Optional)

Any other information that is important to this pull request.

### CODE REVIEW

File: packages/fxa-settings/src/components/App/RememberPassword/en.ftl [hunk 1]
```diff
@@ -0,0 +1 @@
+pw-remember-pw = Remember password? Sign in
```
Comment (bcolsson) [inline packages/fxa-settings/src/components/App/RememberPassword/en.ftl:1]: I know this is migrating a gettext string, but could you add a comment for context? It's not 100% clear that this is a link users can follow to sign in.

Also in the mockup there's other strings, so confirm whether or not those also need to be added as ftl strings.

File: packages/fxa-settings/src/pages/ConfirmResetPassword/en.ftl [hunk 2]
```diff
@@ -0,0 +1,10 @@
+## Confirm Reset Password Component
+
+# Second step of password reset flow for Firefox accounts
+# Header confirming that a password reset email has been sent to the user's email address
+confirm-pw-reset-header = Reset email sent
+
+# Instructions to continue the password reset process
+# ${email} is the email entered by the user and where the password reset instructions were sent
+# "follow" is prefered to "click" or "tap" for better accessibility
+confirm-pw-reset-instructions = Follow the link emailed to ${email} within the next hour to create a new password.
```
Comment (flodolo) [inline packages/fxa-settings/src/pages/ConfirmResetPassword/en.ftl:10]: The variable formatting is not one that I expect to see in Fluent…

```suggestion
confirm-pw-reset-instructions = Follow the link emailed to {$email} within the next hour to create a new password.
```

File: packages/fxa-settings/src/components/App/RememberPassword/en.ftl (hunk 1 above)
Comment (vpomerleau) [reply packages/fxa-settings/src/components/App/RememberPassword/en.ftl:5]: Thank you! Added a comment.

File: packages/fxa-settings/src/pages/ConfirmResetPassword/en.ftl (hunk 2 above)
Comment (vpomerleau) [reply packages/fxa-settings/src/pages/ConfirmResetPassword/en.ftl]: Thank you! I've corrected the variable.

**Review by bcolsson — APPROVED** [review]

**Review by vbudhram — APPROVED** [review]
This LGTM, I've pulled in https://github.com/mozilla/fxa/pull/14701 to get tests passing.

**Comment by vpomerleau** [comment]
I've updated this PR to reflect the changes that had been requested for the ResetPassword page (https://github.com/mozilla/fxa/pull/14599), and this is now ready for review

**Comment by vbudhram** [comment]
This was merged in https://github.com/mozilla/fxa/pull/14701

**Pull request closed without merging**